Tests/FuzzilliTests/LifterTest.swift

```diff
@@ -736,6 +736,30 @@ class LifterTests: XCTestCase {
 
         XCTAssertEqual(lifted_program,expected_program)
     }
+
+    func testRegExpInline(){
+        let fuzzer = makeMockFuzzer()
+        let b = fuzzer.makeBuilder()
+
+        let v0 = b.loadRegExp("a", RegExpFlags(rawValue:0))
```

Comment:
Nit: here and below, I think you can just use `b.loadRegExp("a", RegExpFlags())`